src/component/index/ourService/ourService.js

```diff
@@ -0,0 +1,46 @@
+
+import React, { Component } from 'react';
+
+import podium from './../../../assets/images/podium.png'
+import employee from './../../../assets/images/employee.png'
+import employees from './../../../assets/images/employees.png'
+
+import './style.css';
+
+
+
+class OurService extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {}
+    }
+
+    render() {
+        return (
+            <div className="ourService" >
+                <div className="container-fluid">
+                    <div className="container">
+                        <div className="ours-box" >
+                            <img src={employee} alt="سرویس" />
```

Comment:
icon svg na png! to ghesmate alt="" hame ro nanevis service esteme title service benvis lotfan